MyFitness/MyFitness/Custom Views/ImageViews/NetworkManager.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  NetworkManager.swift
+//  MyFitness
+//
+//  Created by John Kouris on 3/2/20.
+//  Copyright © 2020 John Kouris. All rights reserved.
+//
+
+import UIKit
+
+class NetworkManager {
+    static let shared = NetworkManager()
+    private let baseURL = "https://api.spoonacular.com/recipes/random/"
+    private let apiKey = APIKeys.recipeApiKey
+    let cache = NSCache<NSString, UIImage>()
+    
+    private init() {}
+    
+    func getRecipes(completion: @escaping (Result<RecipeList, MFError>) -> Void) {
+        let queryItems = [URLQueryItem(name: "apiKey", value: apiKey), URLQueryItem(name: "number", value: "30")]
+        var urlComponent = URLComponents(string: baseURL)!
+        urlComponent.queryItems = queryItems
+        
+        guard let url = urlComponent.url else {
+            DispatchQueue.main.async {
```

Comment:
This one doesn't actually need it, because it's not called from the network queue.